Sources/Sentry/SentrySDK.m

```diff
@@ -42,24 +42,32 @@ + (void)setCurrentHub:(SentryHub *)hub {
 }
  
 + (id)initWithOptions:(NSDictionary<NSString *,id> *)optionsDict {
+    [SentrySDK withOptions:optionsDict];
+    return nil;
+}
+
++ (id)initWithOptionsObject:(SentryOptions *)options {
```

Comment:
Does this show up as an overload in swift?

It was suggested the ideal API would be

```swift
SentrySDK.with(options: .init(
    dsn: "https://...”,
    debug: true,
    logLevel: "verbose",
    enableAutoSessionTracking: true,
    sessionTrackingIntervalMillis: 5000
))
```

As opposed to the Dictionary approach (which makes total sense for Objective-C